profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.admin.inc

```diff
@@ -0,0 +1,597 @@
+<?php
+/**
+ * @file
+ * Nexteuropa Newsroom administration page.
+ */
+
+/**
+ * Nexteuropa newsroom admin form.
+ */
+function nexteuropa_newsroom_admin_settings($form, &$form_state) {
+  // Be aware about the selected content types in the fields fieldset.
+  $sel_ct = NewsroomHelper::getFieldInstances('select');
+  $top_ct = NewsroomHelper::getFieldInstances('topic');
+  $newsroom_universe_id = variable_get('newsroom_universe_id', NULL);
+  // Get some information basing on the activation of the universe id.
+  $universe_id = _nexteuropa_newsroom_check_universe_id();
```

Comment:
I have rewritten this part of code. So, we don't call the code twice. Also, I changed the code and improved the calls of the same code.